sdk/core/azure-mgmt-core/azure/mgmt/core/policies/__init__.py

```diff
@@ -47,6 +47,13 @@ class ARMHttpLoggingPolicy(HttpLoggingPolicy):
         "x-ms-request-charge",
     ])
 
+    # Deprecated
+    DEFAULT_HEADERS_WHITELIST = DEFAULT_HEADERS_ALLOWLIST
+
+    def __init__(self, logger=None, **kwargs):
```

Comment:
This fixes a bug in ARMHttpLoggingPolicy. Previously the allowed_header_names would be set to HttpLoggingPolicy.DEFAULT_HEADERS_ALLOWLIST instead of ARMHttpLoggingPolicy.DEFAULT_HEADERS_ALLOWLIST